src/dune_rules/findlib/findlib.ml

```diff
@@ -294,9 +294,11 @@ end = struct
              To workaround this problem, for builtin packages we check that at
              least one of the archive is present. *)
           match archives t with
-          | { byte = []; native = [] } -> Memo.return true
-          | { byte; native } ->
-            Memo.List.exists (byte @ native) ~f:(fun p ->
+          | { byte = []; native = []; melange = [] } -> Memo.return true
+          | { byte; native; melange } ->
+            Memo.List.exists
+              (byte @ native @ melange)
```

Comment:
Is this needed? Findlib doesn't support melange and I doubt that it will.

Reply:
removed in https://github.com/jchavarri/dune/pull/1/commits/36fa79304cf569739829df9c46b8d0213b108aa5.